flask/app/models.py

```diff
@@ -21,6 +22,18 @@ def check_password(self, password):
 
     def get_id(self):
         return self.user_id
+    
+    def make_admin(self):
+        self.is_admin = True
+
+    def revoke_admin(self):
+        self.is_admin = False
```

Comment:
Do these methods serve any purpose or will they be expanded to do something?

Reply:
I figured we'd need the ability to edit someone's admin status. Now that I'm looking through though, this would probably be better suited for the routes.py file?

Reply:
If these don't add additional functionality related to the model like `set_password` does then they don't have a purpose.